src/rdb/rdb_raft.c

```diff
@@ -2020,12 +2032,13 @@ rdb_raft_append_apply(struct rdb *db, void *entry, size_t size, void *result)
 int
 rdb_raft_verify_leadership(struct rdb *db)
 {
+	if (raft_has_majority_leases(db->d_raft))
+		return 0;
 	/*
-	 * raft does not provide this functionality yet; append an empty entry
-	 * as a (slower) workaround.
-	 */
-	return rdb_raft_append_apply(db, NULL /* entry */, 0 /* size */,
-				     NULL /* result */);
+	 * Since raft does not provide a function for verifying leadership via
+	 * RPCs yet, append an empty entry as a (slower) workaround.
+ 	 */
```

Comment:
(style)  code indent should use tabs where possible
(style)  please, no space before tabs